tests/unittest_manager.py

```diff
@@ -111,6 +110,16 @@ def test_ast_from_namespace_pkgutil(self) -> None:
     def test_ast_from_namespace_pkg_resources(self) -> None:
         self._test_ast_from_old_namespace_package_protocol("pkg_resources")
 
+    def test_identify_old_namespace_package_protocol(self) -> None:
+        # Like the above cases, this package follows the old namespace package protocol
+        # astroid currently assumes such packages are in sys.modules, so import it
+        # pylint: disable-next=import-outside-toplevel
+        import tests.testdata.python3.data.path_pkg_resources_1.package.foo as _  # noqa
+
+        self.assertTrue(
+            util.is_namespace("tests.testdata.python3.data.path_pkg_resources_1")
+        )
```

Comment:
```suggestion
        assert util.is_namespace("tests.testdata.python3.data.path_pkg_resources_1")
```

But this is very personal preference-y.

Reply:
Yeah in pytest that's better, but I guess since this one inherits from unittest I wanted to keep it this way. `unittest` output gets mangled when you do a bare assert. When those decorators for the "cachesetupmixin" go away, then maybe this doesn't need to inherit from unittest anymore.